app/src/main/res/xml/rowkeys_turkish_q1.xml

```diff
@@ -0,0 +1,178 @@
+** You may obtain a copy of the License at
+**
+**     http://www.apache.org/licenses/LICENSE-2.0
+**
+** Unless required by applicable law or agreed to in writing, software
+** distributed under the License is distributed on an "AS IS" BASIS,
+** WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+** See the License for the specific language governing permissions and
+** limitations under the License.
+*/
+-->
+
+<merge
+    xmlns:latin="http://schemas.android.com/apk/res-auto"
+>
+    <switch>
+        <case
+            latin:showNumberRow="true"
+            latin:showExtraChars="true">
+            <Key
+                latin:keySpec="!text/keyspec_q"
+                latin:keyHintLabel="_"
+                latin:additionalMoreKeys="_"
+                latin:moreKeys="!text/morekeys_q" />
+            <Key
+                latin:keySpec="!text/keyspec_w"
+                latin:keyHintLabel="\\"
+                latin:additionalMoreKeys="\\\\"
+                latin:moreKeys="!text/morekeys_w" />
+            <Key
+                latin:keySpec="e"
+                latin:keyHintLabel="|"
+                latin:additionalMoreKeys="\\|"
+                latin:moreKeys="!text/morekeys_e" />
+            <Key
+                latin:keySpec="r"
+                latin:keyHintLabel="="
+                latin:additionalMoreKeys="="
+                latin:moreKeys="!text/morekeys_r" />
+            <Key
+                latin:keySpec="t"
+                latin:keyHintLabel="["
+                latin:additionalMoreKeys="!text/keyspec_left_square_bracket"
+                latin:moreKeys="!text/morekeys_t" />
+            <Key
+                latin:keySpec="!text/keyspec_y"
+                latin:keyHintLabel="]"
+                latin:additionalMoreKeys="!text/keyspec_right_square_bracket"
+                latin:moreKeys="!text/morekeys_y" />
+            <Key
+                latin:keySpec="u"
+                latin:keyHintLabel="&lt;"
+                latin:additionalMoreKeys="!text/keyspec_less_than"
+                latin:moreKeys="!text/morekeys_u" />
+            <Key
+                latin:keySpec="ı"
+                latin:keyHintLabel="&gt;"
+                latin:additionalMoreKeys="!text/keyspec_greater_than"
+                latin:moreKeys="!text/morekeys_i" />
+            <Key
+                latin:keySpec="o"
+                latin:keyHintLabel="{"
+                latin:additionalMoreKeys="!text/keyspec_left_curly_bracket"
+                latin:moreKeys="!text/morekeys_o" />
+            <Key
+                latin:keySpec="p"
+                latin:keyHintLabel="}"
+                latin:additionalMoreKeys="!text/keyspec_right_curly_bracket" />
+            <Key
+                latin:keySpec="ğ"
+                latin:moreKeys="!text/morekeys_g" />
+            <Key
+                latin:keySpec="ü"
+                latin:moreKeys="!text/morekeys_u" />
+        </case
+            latin:showNumberRow="true">
```

Comment:
This line has invalid token

Reply:
i don't know which one is invalid, maybe the empty case tag is the problem?